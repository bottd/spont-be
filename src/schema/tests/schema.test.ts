import { schema } from '../';

describe('Schema', () => {
  it('should match snapshot', () => {
    expect(schema).toMatchSnapshot();
  });
});
