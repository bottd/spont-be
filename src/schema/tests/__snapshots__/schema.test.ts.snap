// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Schema should match snapshot 1`] = `
GraphQLSchema {
  "__allowedLegacyNames": Array [],
  "__validationErrors": undefined,
  "_directives": Array [
    "@include",
    "@skip",
    "@deprecated",
  ],
  "_implementations": Object {},
  "_mutationType": "RootMutation",
  "_possibleTypeMap": undefined,
  "_queryType": "RootQuery",
  "_subscriptionType": undefined,
  "_typeMap": Object {
    "Boolean": "Boolean",
    "Coordinate": "Coordinate",
    "Float": "Float",
    "Location": "Location",
    "RootMutation": "RootMutation",
    "RootQuery": "RootQuery",
    "String": "String",
    "User": "User",
    "__Directive": "__Directive",
    "__DirectiveLocation": "__DirectiveLocation",
    "__EnumValue": "__EnumValue",
    "__Field": "__Field",
    "__InputValue": "__InputValue",
    "__Schema": "__Schema",
    "__Type": "__Type",
    "__TypeKind": "__TypeKind",
  },
  "astNode": undefined,
  "extensionASTNodes": undefined,
}
`;
